src/core/components/courses/Trello.vue

```diff
@@ -22,7 +22,7 @@ export default {
       return [
         { title: 'À venir', courses: this.courseListForthcoming },
         { title: 'En cours', courses: this.courseListInProgress },
-        { title: 'Terminée(s)', courses: this.courseListCompleted },
+        { title: 'Terminées', courses: this.courseListCompleted },
```

Comment:
En vrai c'est juste syntaxiquement mais pourquoi ne pas utiliser ta fonction et this.courseListCompleted.length() ?

Reply:
dans le titre de la section, on peut garder le s => l'idee derriere c'est : dans cette colonne on affiche les formations terminées

Reply:
Je pense qu'on veux pa afficher le nbr de formations. Je pense ici qu'on a pas besoin du (s) comme on a pas besoin dans mettre un dans le titre 'Formations' de cette meme page (ni management courses) et dans tous les répertoires, ou sections qui contiennent plusieurs items. 
Qu'il y en ai 0, 1 ou plus ca reste l'endroit spécifique ou mettre LES formations terminées, comme l'endroit DES poubelles ou le placard DES confitures. 
T'en pense quoi

Reply:
Je suis d'accord